.github/workflows/stale-issue-pr.yml

```diff
@@ -0,0 +1,50 @@
+name: Close inactive issues
```

Comment:
Will this affect existing issues? We probably have some old stale ones.

Reply:
This applies only when the label "stat:awaiting response from contributor" is applied